src/components/LoaderComponent.jsx

```diff
@@ -1,27 +1,32 @@
 import React from 'react';
-import { Dimmer, Loader, Segment } from 'semantic-ui-react';
+import { Container, Dimmer, Grid } from 'semantic-ui-react';
 import PropTypes from 'prop-types';
 
-const LoaderComponent = ({ loadingText, size, showDimmer, dimmerStyle }) => (
-  <Segment>
-    <Dimmer active={showDimmer} style={dimmerStyle}>
-      <Loader inverted size={size}>{loadingText}</Loader>
+import '../_css/LoaderComponent.css';
+
+const LoaderComponent = ({ loadingText }) => (
+  <Container>
+    <Dimmer active inverted className="overlay">
+      <Grid centered columns={3}>
+        <Grid.Column className="loader-column">
+          <div className="la-square-jelly-box la-2x">
+            <div />
```

Comment:
Are the empty `<div />` necessary? Can't padding, margin or even `:before` and `:after` work?

Reply:
Based on how the source code for the loader is structured, the two divs are necessary.
Implementing `:before` and `:after` will mean having to rewrite code that already exists in the `loader.min.css` file